N2A/src/gov/sandia/n2a/ui/settings/SettingsRepo.java

```diff
@@ -179,6 +178,8 @@ public class SettingsRepo extends JScrollPane implements Settings
     public SettingsRepo ()
     {
         instance = this;
+        
+        AppData.documents.forEach (c -> existing.put (c.key (), ((MCombo) c).getContainerMap ()));
```

Comment:
It would be a little more consistent with MNode usage to say
for (MNode c : AppData.documents) existing.put (...);